Control/lib/services/StatusService.js

```diff
@@ -179,23 +179,23 @@ class StatusService {
   }
 
   /**
-   * Build a response containing the information and status of the Kafka Service
+   * Build a response containing the information and status of the Notification Service
    * @return {Promise<Resolve>}
    */
-  async getKafkaStatus(kafkaConnector) {
-    let kafka = {};
-    if (kafkaConnector.isConfigured()) {
-      kafka = this.config.kafka;
+  async getNotificationStatus(notification) {
```

Comment:
Even though it was missed before, could you please add documentation for the type of parameter received?